app/src/test/java/com/findmyrecycling/ProductTests.kt

```diff
@@ -147,7 +147,7 @@ class ProductTests {
         assertTrue(allProducts!!.isNotEmpty())
         var containsCellPhone = false
         allProducts!!.forEach {
-            if (it.product.equals("Cell Phone")) {
+            if (it.product == "Cell Phone") {
```

Comment:
simplified operator

Reply:
This was covered in a previously submitted review. When that occurs, first in wins.